README.md

```diff
@@ -0,0 +1,9 @@
+# Universal Packaging Tool: MacPorts backend
+This is a [MacPorts](https://www.macports.org) backend for [upt](https://pypi.python.org/pypi/upt).
+
+#### Steps to use this tool out:
+
+- Install upt
```

Comment:
From what I gather the `python3 setup.py install` commands already installs `upt`, so this one might not be needed? But I assume one does need to install `upt-pypi`? I would add the instruction `pip install upt-pypi` after the one for installing from this repository.

Reply:
@mojca I guess it's just `pip install upt`: https://pypi.org/project/upt/